bikeshed/metadata.py

```diff
@@ -334,6 +345,35 @@ def parseDate(key, val, lineNum):
         return None
 
 
+def parseDateOrDuration(key, val, lineNum):
```

Comment:
Also accept "never", returning None, so you can override a default expiry. Maybe allow a boolish false too?

Reply:
Added never and boolish false in a33f871.